src/Lecture1.hs

```diff
@@ -119,4 +145,18 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+getNums :: Int -> [Int] -> [Int] -> [Int]
+getNums n result list
+    | null list = result
+    | ((head list) > n) = getNums n ((head result + 1):[] ++ (tail result)) (tail list)
+    | (head list) == n = getNums n result (tail list)
+    | (head list) < n = getNums n (init result ++ (last result + 1):[]) (tail list)
+
+
+getOutput :: Int -> [Int]-> [Char] 
+getOutput n list = 
+    show n ++ " is greater than "++ show (last list) ++ " elements and lower than "++ show (head list) ++ " elements"
+
+lowerAndGreater :: Int -> [Int] -> [Char]
+lowerAndGreater n list =
+    getOutput n (getNums n [0,0] list)
```

Comment:
I see that your solution can be optimized. It looks like you collect a list of counts. But I don't think you need a list. You can have two variables of type `Int` where you can count how many numbers are lower and greater.